v2/commands/job/logs.go

```diff
@@ -67,45 +73,71 @@ func Logs(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Error: unable to write header")
 	}
 
-	for {
-		// write rows as the batches come in up to the limit
-		if limit > 0 && limit <= total {
-			break
-		}
-
-		res, err := logs(ctx, runner.Client, p.ProjectId, p.TaskId, jobId, pageIndex)
-		if err != nil {
-			return err
-		}
-
-		currentPage := len(res.Items)
-		if total+currentPage >= limit {
-			currentPage = limit - total
-		}
+	latestLogTime := time.Time{}
```

Comment:
it's very slightly more idiomatic for this to be:
```suggestion
	var latestLogTime time.Time
```